src/copairs/compute.py

```diff
@@ -4,15 +4,16 @@
 import os
 from multiprocessing.pool import ThreadPool
 from pathlib import Path
-from typing import Callable, Tuple, Union, Optional
+from typing import Callable, Optional, Tuple, Union
 
 import numpy as np
-from tqdm.autonotebook import tqdm
-from scipy.spatial.distance import cdist
 from scipy.spatial.distance import _METRICS_NAMES as SCIPY_METRICS_NAMES
+from scipy.spatial.distance import cdist
 
 
-def parallel_map(par_func: Callable[[int], None], items: np.ndarray) -> None:
+def parallel_map(
+    par_func: Callable[[int], None], items: np.ndarray, progress_bar: bool = True
```

Comment:
`parallel_map` is used in other places, but the call signature wasn't updated. To make things consistent, this change needs to be propagated upstream, so the progress bar can also be disabled when e.g. `average_precision` is called.